outputs.tf

```diff
@@ -14,51 +14,50 @@
  * limitations under the License.
  */
 
-output "keyring" {
-  description = "Self link of the keyring."
-  value       = google_kms_key_ring.key_ring.id
-
-  # The grants are important to the key be ready to use.
-  depends_on = [
-    google_kms_crypto_key_iam_binding.owners,
-    google_kms_crypto_key_iam_binding.decrypters,
-    google_kms_crypto_key_iam_binding.encrypters,
-  ]
+output "existing_keyring" {
+  description = "Existing keyring is used, i.e. keyring has been created."
+  value       = var.existing_keyring
```

Comment:
This does not seem very useful since its just plumbing input.

Reply:
So the output has the complete overall state if sourced.

Reply:
Users generally invoke the module in a root config, so if they want to plumb static values they can do so in their root module so we don't pollute this module's interface. Lets remove these.